DQM/BeamMonitor/test/Online_BeamMonitor_file.py

```diff
@@ -0,0 +1,84 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("DQM")
+process.load("FWCore.MessageLogger.MessageLogger_cfi")
+
+
+#ESProducer                                                                                                                                                                                                      
+process.load("CondCore.CondDB.CondDB_cfi")                                                                                                                                                                       
+process.BeamSpotDBSource = cms.ESSource("PoolDBESSource",                                                                                                                                                        
+                       process.CondDB,                                                                                                                                                                          
```

Comment:
```suggestion
process.load("CondCore.CondDB.CondDB_cfi")
process.BeamSpotDBSource = cms.ESSource("PoolDBESSource",
                                        process.CondDB,
                                        toGet = cms.VPSet(
                                            cms.PSet(
                                                record = cms.string('BeamSpotOnlineLegacyObjectsRcd'),
                                                tag = cms.string("BeamSpotOnlineTestLegacy"),
                                                refreshTime = cms.uint64(1)
                                            ),
                                            cms.PSet(
                                                record = cms.string('BeamSpotOnlineHLTObjectsRcd'),
                                                tag = cms.string("BeamSpotOnlineTestHLT"),
                                                refreshTime = cms.uint64(1)

                                            ),
                                ),
                                        )
process.BeamSpotDBSource.connect = cms.string('frontier://FrontierProd/CMS_CONDITIONS') 
process.BeamSpotESProducer = cms.ESProducer("OnlineBeamSpotESProducer")


```

Reply:
I am not sure I get the meaning of the proposal it seems to me it is identical to the original one. Is it just only indentation inside the parameter definition of PoolDBESSource ?